### ISSUE TITLE
Hacktoberfest participation

### ISSUE THREAD

**Issue by CollinAlpert** [dotnet/roslyn-analyzers#6988]
I wanted to inquire about this repository's position on Hacktoberfest. I see that dotnet/roslyn and dotnet/runtime have opted-in to Hacktoberfest, yet this repository has not. Is this a conscious choice?

**Comment by mavasani** [comment]
@arkalyanms

**Comment by arkalyanms** [comment]
No, its just that the repo has never had any associated topics created before and when the hacktoberfest topic came to be required, it probably  got missed. Should be added now.
